app/views/admin/travel_insurance_firms/_extra_sections.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if this_is_a_travel_insurance_firm then %>
+  <div class="t-registration-questions">
+    <div>
+      <h2>Travel Registration Questions</h2>
+    </div>
+    <% travel_registration_questions.each do | question |%> 
+      <div class="row">
+        <div class="l-2col-side--align-right col-xs-6">
+          <%= question %>
```

Comment:
Can we get the question or scenario details from the locale file so it's more understandable to the user? Alternative is to use `question.humanize` but if we can do something with locale it would be better

Reply:
I'm afraid not. Both options were implemented and shown to Dobrin. He prefers the symbol to the full on text. Even more so because of questions like "Please chose one of the options below". It makes the Admin's job easier.

Reply:
ok then 👍